debug/Dockerfile

```diff
@@ -16,11 +16,21 @@ RUN apk add --no-cache \
 RUN mkdir -p -m 0600 ~/.ssh && ssh-keyscan github.com >> ~/.ssh/known_hosts
 
 # Install https://github.com/ellistarn/home
-RUN --mount=type=ssh sh -c \
-    "$(wget https://raw.githubusercontent.com/ellistarn/home/main/bin/install-home.sh -O -)"
+#RUN --mount=type=ssh sh -c \
+#   "$(wget https://raw.githubusercontent.com/ellistarn/home/main/bin/install-home.sh -O -)"
 
 RUN --mount=type=ssh sh -c \
     "$(wget https://raw.githubusercontent.com/ellistarn/home/main/bin/install-dependencies.sh -O -)"
 
 # Set default shell to Zsh
 RUN sed -i 's|root:x:0:0:root:/root:/bin/ash|root:x:0:0:root:/root:/bin/zsh|' /etc/passwd
+
+RUN apk add --no-cache \
```

Comment:
Nice! Can you add a comment above:
# Install AWS CLI

Reply:
sure thing